silverback/_cli.py

```diff
@@ -26,6 +27,20 @@
 from silverback.runner import PollingRunner, WebsocketRunner
 from silverback.worker import run_worker
 
+from .exceptions import ImportFromStringError
+
+DOCKERFILE_CONTENT = """
+FROM ghcr.io/apeworx/silverback:latest
```

Comment:
This should probably be the suggested default

```suggestion
FROM ghcr.io/apeworx/silverback:stable
```